lib/slipstream/command_router.ex

```diff
@@ -30,6 +32,10 @@ defmodule Slipstream.CommandRouter do
   @spec route_command(struct()) :: any()
   def route_command(command)
 
+  def route_command(%OpenConnection{config: %Config{test_mode?: true}}) do
+    :ok
+  end
```

Comment:
if a connection is started in `test_mode?: true`, we do _not_ spawn a connection process for it, so there's no chance that it will attempt to do any network activity